addons/sale/models/sale_order.py

```diff
@@ -11,6 +11,8 @@
 from odoo.tools import float_is_zero, format_amount, format_date, html_keep_url, is_html_empty
 from odoo.tools.sql import create_index
 
+import json
```

Comment:
Order of import it should be in first section [here](https://github.com/odoo/odoo/pull/152270/files#diff-695321b9df7129604a59fbc9bff34f97ba90b3394aa1166ca24e36b27389ae82R5)